contrib/rpcimportable/go.mod

```diff
@@ -18,7 +18,7 @@ replace (
 )
 
 replace (
-	github.com/cosmos/evm => github.com/zeta-chain/evm v0.0.0-20250807160911-14d7b1cca905
+	github.com/cosmos/evm => github.com/zeta-chain/evm v0.0.0-20250808111716-1882abec3ec9
```

Comment:
Can't we use a tag?

Reply:
we were using commits for ethermint as well, not sure which tag we would put there since we are using cosmos evm main branch

Reply:
Isn't it generally synchronized with the v0.3.1 release of cosmos/evm?